inception-project/inception
# #3140 - Allow basic theming

**What's in the PR**
- Added basic theming support including basic documentation
- Incorporated inception.css into the custom bootstrap theme
- Rename webanno-XXX.scss to inception-XXX.scss

**How to test manually**
* Place a `theme.css` file with custom CSS rules into the application home folder
* See documentation

**Automatic testing**

**Documentation**
